modules/tinymce/src/core/main/ts/api/dom/Selection.ts

```diff
@@ -100,7 +100,7 @@ interface EditorSelection {
     parents: Element[];
   }) => void) => { unbind: () => void };
   getScrollContainer: () => HTMLElement;
-  scrollIntoView: (elm: Element, alignToTop?: boolean) => void;
+  scrollIntoView: (elm?: HTMLElement, alignToTop?: boolean) => void;
```

Comment:
Should this wait until 6.0, for semver reasons? Technically code that used to compile will no longer compile with this change

Reply:
Code that didn't pass a `HTMLElement` previously also would have thrown an exception (since it wouldn't have been able to read the dimensions), so no this is simply a bugfix.

Reply:
Can we add a changelog line saying "fixed an incorrect type signature where `Element` could be passed into `editor.selection.scrollIntoView`" or something?

Reply:
Point... we should have that. Anyways, this change isn't really needed so if we'd prefer I'm happy to revert it (I just figured you would likely pick it up and raise it as an issue to be honest).

Reply:
I'm fine with either reverting or adding a changelog entry. It's always nice to move our type signatures closer to how the editor actually behaves, so I'm quite happy to keep it in.